### PR TITLE
Update benchmark tool to scrape metrics from JSON files

### PR DESCRIPTION
**Pull request by shyamjvs**
Following from recent changes of moving all metrics data from build-log.txt to individual JSON files.

Fun fact: There are literally just 6 godeps now! Turns out all the evil was in `k8s.io/kubernetes/test/e2e/framework` package which was importing the universe.

cc @wojtek-t @gmarek

### CODE REVIEW

**Comment by gmarek** [comment]
And we deleted 1.5m lines. I can't really tell how happy I am about that:)

**Comment by shyamjvs** [comment]
Some more stats to make you happier:
- Size of the directory reduced from 69MB to 6.6MB
- Compile time reduced from 44s to 8s
- Time to run tests reduced from 3m50s to 2.5s (CPU time)

Moral: Importing `k8s.io/kubernetes/test/e2e/framework` can be harmful for your health.

File: benchmark/pkg/metricsfetcher/scraper/scraper.go [hunk 1]
```diff
@@ -17,87 +17,88 @@ limitations under the License.
...
-				buff.Reset()
-			} else if strings.Contains(line, apiLatencyResultTag) {
-				state = readingAPICallLatencies
-				buff.WriteString("{ ")
-			} else if strings.Contains(line, podLatencyResultTag) {
-				state = readingPodStartupLatency
-				buff.WriteString("{ ")
+// GetMetricsFilesForRun for a given run of a job, returns a map of testname ("load", "density", etc) to a
+// list of paths to its latency files (API responsiveness, pod startup) relative to the run's root dir.
+func GetMetricsFilesForRun(job string, run int, utils util.JobLogUtils) map[string][]string {
```
Comment (gmarek) [inline benchmark/pkg/metricsfetcher/scraper/scraper.go]: GetMetricFilePathsForRun

**Comment by gmarek** [comment]
One naming nit, lgtm otherwise.

File: benchmark/pkg/metricsfetcher/scraper/scraper.go (hunk 1 above)
Comment (shyamjvs) [inline benchmark/pkg/metricsfetcher/scraper/scraper.go]: Done.

**Comment by shyamjvs** [comment]
Adding lgtm based on above comment.

**Pull request merged**